modules/microshift-info-collected-telemetry.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * microshift_support/microshift-remote-cluster-monitoring.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="microshift-info-collected-by-telemetry_{context}"]
+= Information collected by the {microshift-short} Telemetry API
+
+All metrics combined are generally under 2KB and not expected to consume cluster resources.
+
+The following information is collected by Telemetry:
+
+[id="microshift-telemetry-system-information_{context}"]
+== System information
+
+The system information describes the basic configuration of your {microshift-short} cluster and where it is running, for example:
+
+* Version information, including the {microshift-short} cluster version.
+* The {op-system-base-full} version.
+* The {op-system-base-full} deployment type.
+
+[id="microshift-telemetry-sizing-information_{context}"]
+== Sizing information
+
+Sizing information details the cluster capacity, for example:
+
+* The CPU cores {microshift-short} can use.
+* Architecture information.
+* The usable bytes of memory.
+
+[id="microshift-telemetry-usage-information_{context}"]
+== Usage information
+
+Usage information outlines what is happening in the cluster, for example:
+
+* The CPU usage in percentage.
+* The memory usage in percentage.
+* The number of Kubernetes objects by resource type (CRDs).
+* The number of running containers, namespaces, and running pods.
+* The number of routes, ingress, services.
+
+[NOTE]
+====
+Telemetry does not collect identifying information such as usernames or passwords. Red{nbsp}Hat does not intend to collect personal information. If Red{nbsp}Hat discovers that personal information has been inadvertently received, Red{nbsp}Hat deletes such information. To the extent that any telemetry data constitutes personal data, refer to the link:https://www.redhat.com/en/about/privacy-policy[Red{nbsp}Hat Privacy Statement] for more information about Red{nbsp}Hat's privacy practices.
```

Comment:
"For more information about Red{nbsp}Hat's privacy practices, see link:https://www.redhat.com/en/about/privacy-policy[Red{nbsp}Hat Privacy Statement]."

Reply:
This seems like legacy legalese to me, so I'm going to leave it alone.